packages/angular/cli/src/analytics/analytics-collector.ts

```diff
@@ -62,8 +61,8 @@ export class AnalyticsCollector {
       [UserCustomDimension.OsArchitecture]: os.arch(),
       // While User ID is being collected by GA, this is not visible in reports/for filtering.
       [UserCustomDimension.UserId]: userId,
-      [UserCustomDimension.NodeVersion]: nodeVersion,
-      [UserCustomDimension.NodeMajorVersion]: +nodeVersion.split('.', 1)[0],
+      [UserCustomDimension.NodeVersion]: nodeVersion ?? 'other',
+      [UserCustomDimension.NodeMajorVersion]: nodeVersion ? +nodeVersion.split('.', 1)[0] : 'other',
```

Comment:
This parameter can only have a value as a number or undefined.

Reply:
Updated to be `undefined` if we don't have a valid version.